find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
```

Comment:
it's not necessary to declare max at this point since it's declared on line 9 as well.